aif360/sklearn/datasets/openml_datasets.py

```diff
@@ -228,7 +235,12 @@ def fetch_bank(*, data_home=None, cache=True, percent10=False, usecols=None,
             df[col] = df[col].cat.remove_categories('unknown')
     df.education = df.education.astype('category').cat.reorder_categories(
         ['primary', 'secondary', 'tertiary'], ordered=True)
+    
+    # binarize protected attribute (but not corresponding feature)
+    age = (pd.cut(df.age, [0, 25, 60, 100],
+                  labels=False if numeric_only else ['unpriv', 'priv', 'unpriv'])
```

Comment:
Why do we need to order? If we order the age column, it will not match the df anymore for later usage, correct?

Reply:
Ordering the categories is useful as a way to keep track of which group is privileged. This way, we can use `X.age.cat.codes` to convert the categorical column to binary (integers) and we know the privileged class will translate to 1 and unprivileged to 0. Otherwise, I think it just defaults to alphabetical order.

It doesn't shuffle the order of the samples in the dataframe; it's just a flag in the dtype.

Reply:
Thanks for the explanation! Reordering is now implemented